frontend/packages/knative-plugin/integration-tests/features/eventing/eventing-broker-actions.feature

```diff
@@ -0,0 +1,123 @@
+        Scenario: Sidebar for the Broker: KC-01-TC02
+             When user clicks on "default" broker to open the sidebar
+             Then user will see "default" as title of broker
+              And user will see the Resources tab
+              And user will see the Details tab
+
+        
+        @regression @to-do @odc-5030
+        Scenario: Details tab for the Broker: KC-01-TC03
+             When user clicks on "default" broker to open the sidebar
+              And user selects Details tab
+             Then user will see name of broker as "default"
+              And user will see namespace of broker as "aut-eventing-broker-actions"
+              And user will see labels and annotations associated with broker
+              And user will see the owner and broker created time
+
+
+        @regression @to-do @odc-5030
+        Scenario: Edit Application grouping to new application action on Broker: KC-01-TC04
+             When user right clicks on "default" broker to open the context menu
+              And user clicks on "Edit Application grouping"
+              And user will click on Application drop down on the modal
+              And user selects "Create Application" from Application drop down
+              And user enters application name as "new-app"
+              And user clicks on Save button
+             Then user will see "new-app" application created
+
+        
+        #Please add the label as test data on line 59 and verify it on line 63
+        @regression @to-do @odc-5030
+        Scenario: Edit Labels action on Broker: KC-01-TC05
+             When user right clicks on "default" broker to open the context menu
+              And user clicks on "Edit labels"
+              And user adds new label
+              And user clicks on the Save button on the modal
+              And user clicks on "default" Broker to open the sidebar
+              And user selects Details tab
+             Then user will see the newly added label
+
+
+        #Please add the label as test data on line 71 and verify it on line 75
+        @regression @to-do @odc-5030
+        Scenario: Edit Annotations action on Broker: KC-01-TC06
+             When user right clicks on "default" broker to open the context menu
+              And user clicks on "Edit annotations"
+              And user adds new annotations
+              And user clicks on the Save button on the modal
+              And user clicks on "default" broker to open the sidebar
+              And user selects Details tab
+             Then user will see the newly added annotations
+
+
+        @regression @to-do @odc-5030
+        Scenario: Edit Broker action on Broker: KC-01-TC07
+             When user clicks on "default" Broker to open the sidebar
+              And user selects "Edit Broker" from Actions drop down
+             Then user will see the YAML editor to edit the Broker
+              And user will see Details and Triggers tab
```

Comment:
IMO you need to edit something in YAML and check its effect on the broker for this scenario

Reply:
Then I have to make this as a manual scenario @sanketpathak

Reply:
Updated the scenario and made it manual